Extensions/PanelSpriteObject/panelspriteruntimeobject.js

```diff
@@ -59,38 +59,71 @@ gdjs.PanelSpriteRuntimeObject.prototype.extraInitializationFromInitialInstance =
     }
 };
 
+/**
+ * Set the x position of the panel sprite.
+ * @param {number} x The new x position in pixels.
+ */
 gdjs.PanelSpriteRuntimeObject.prototype.setX = function(x) {
     gdjs.RuntimeObject.prototype.setX.call(this, x);
     this._renderer.updatePosition();
 };
 
+/**
+ * Set the y position of the panel sprite.
+ * @param {number} y The new y position in pixels.
+ */
 gdjs.PanelSpriteRuntimeObject.prototype.setY = function(y) {
     gdjs.RuntimeObject.prototype.setY.call(this, y);
     this._renderer.updatePosition();
 };
 
+/**
+ * Set the texture of the panel sprite.
+ * @param {string} textureName The name of the texture.
+ * @param {RuntimeScene} runtimeScene The scene the object lives in.
```

Comment:
I think you need to specify `{gdjs.RuntimeScene}`?

Reply:
What do you mean?

Reply:
Yes I think it's required to have the "fully qualified" name: `gdjs.RuntimeScene` and not simply `RuntimeScene` (which would be a global class, while `RuntimScene` leaves inside `gdjs` object).